common/src/main/java/com/lightstep/tracer/shared/ClientMetrics.java

```diff
@@ -4,21 +4,21 @@
 import com.lightstep.tracer.grpc.MetricsSample;
 
 import java.util.ArrayList;
-import java.util.concurrent.atomic.AtomicLong;
+import java.util.concurrent.atomic.AtomicInteger;
```

Comment:
Why the change to integer?

Reply:
We hopefully don't drop as many Spans that we require a `long` :) But more importantly, this was creating warnings all over the place as we had been bad at keeping uniformity for this very metric (in some places we used int, in some other ones long, etc).